dockerimage/popscle/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# based on the dockerfile from https://github.com/statgen/popscle
+
+# Source Image
+FROM ubuntu:18.04
```

Comment:
hey, thanks for your suggestion. I have now specified the docker configuration directly in the config file. To avoid any misunderstanding, I will delete images filer under the dockerimage folder.
popscle works on the newest ubuntu, will update it.